awesomemotive/easy-digital-downloads
# Show quantities for existing orders which have them

Fixes #8281

Proposed Changes:
1. If an existing order has quantities of an item, show that in the order details, refund details, and refund modal, even if quantities have since been disabled.

To test:
1. Enable quantities.
2. Place an order with multiples of at least one item.
3. Disable quantities.
4. The order will not show quantities in `release/3.0`, but in this branch, quantities should show on the order details screen, refund modal, and refund details screen.
